app/assets/javascripts/components/header.es6.jsx

```diff
@@ -2,13 +2,13 @@ class Header extends Component {
 
   static get propTypes() {
     return {
-      handleSidebarClick: React.PropTypes.func.isRequired,
+      handleClick: React.PropTypes.func.isRequired,
```

Comment:
I just didn't like the func name `handleSidebarClick` so for now we're defaulting to a more general name.